speech-assets/UtterancesTemplate.txt

```diff
@@ -203,6 +203,21 @@ AggregationIntent {Aggregate} (entries in|rows in) {TableName} \
   {ComparisonColumnThree} {ComparatorThree} [{ColumnValueThree}|{ColumnNumberThree}] \
   {GroupBy} {GroupByColumn}
 
+AggregationIntent {Aggregate} the {AggregationColumn} (column) in {TableName} \
+  (table) \
+  where {ComparisonColumnOne} {ComparatorOne} [{ColumnValueOne}|{ColumnNumberOne}] \
+  {BinaryLogicOpOne} \
+  {ComparisonColumnTwo} {ComparatorTwo} [{ColumnValueTwo}|{ColumnNumberTwo}] \
+  {BinaryLogicOpTwo} \
+  {ComparisonColumnThree} {ComparatorThree} [{ColumnValueThree}|{ColumnNumberThree}] \
+  {GroupBy} {GroupByColumn}
```

Comment:
confused about why this is listed as an addition. I definitely didn't add this. Any idea?